addons/binding/org.openhab.binding.yamahareceiver/src/main/java/org/openhab/binding/yamahareceiver/internal/protocol/xml/XMLConstants.java

```diff
@@ -0,0 +1,39 @@
+/**
+ * Copyright (c) 2010-2018 by the respective copyright holders.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.openhab.binding.yamahareceiver.internal.protocol.xml;
+
+import org.openhab.binding.yamahareceiver.YamahaReceiverBindingConstants.Feature;
+
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * XML protocol constants.
+ *
+ * @author Tomasz Maruszak - Initial contribution
+ */
+public class XMLConstants {
+
+    public static final String ON = "On";
+    public static final String OFF = "Off";
+    public static final String POWER_STANDBY = "Standby";
+    public static final Map<String, Feature> FEATURE_BY_YNC_TAG;
+
+    static {
```

Comment:
I personally dislike this construct, can you just call a method which returns the complete list, now it can have all values in between in a multi-threaded env

Reply:
I wanted to avoid Guava to help with map initialization and also wanted to have just one map instance.
Static class constructors are executed thus are [thread safe](https://stackoverflow.com/a/878624/1906057), so it should be good. If it's a matter of preference I could change it to a method call (or one line in guava).

Reply:
> Static class constructors are executed once per class loader thus are thread safe, 

What a great language :-)

Lets leave it then, if there is no reason for change except my personal preference :-)